great_expectations/experimental/metric_repository/metrics.py

```diff
@@ -74,14 +74,16 @@ class ColumnMetric(Metric, Generic[_ValueType]):
 # Metrics with parameters (aka metric_value_kwargs)
 # This is where the concrete metric types are defined that
 # bring together a domain type, value type and any parameters (aka metric_value_kwargs)
+# If a metric has parameters, it should be defined here. If it doesn't, you can use the generic types above, for
+# example, ColumnMetric[float] or TableMetric[list[str]].
 
 # TODO: Add metrics here for all Column Descriptive Metrics
-#  QuantileValuesColumnMetric is an example of a metric that has parameters
+#  ColumnQuantileValuesMetric is an example of a metric that has parameters
```

Comment:
Just confirming: this is where we are bringing the naming consistent with the domain models in Mercury?

Reply:
Yep!